e2e/cypress/integration/system_console/search_box_spec.js

```diff
@@ -0,0 +1,120 @@
+};
+
+describe('System console', () => {
+    before(() => {
+        cy.apiRequireLicense();
+    });
+
+    it('MM-T897_1 - Focus should be in System Console search box on opening System Console or refreshing pages in System Console', () => {
+        const pageIds = ['reporting\\/system_analytics', 'reporting\\/team_statistics', 'reporting\\/server_logs', 'user_management\\/users', 'user_management\\/teams'];
+        goToAdminConsole();
+
+        // * Assert the ID of the element is the ID of admin sidebar filter
+        cy.focused().should('have.id', 'adminSidebarFilter');
+        cy.wait(TIMEOUTS.ONE_SEC);
+
+        pageIds.forEach((id) => {
+            // # Go to another page
+            cy.get(`#${id}`).click();
+
+            // * Ensure focus is lost
+            cy.focused().should('not.have.id', 'adminSidebarFilter');
+
+            // * Reload and ensure the focus is back on the search component
+            cy.reload();
+            cy.focused().should('have.id', 'adminSidebarFilter');
+            cy.wait(TIMEOUTS.ONE_SEC);
+        });
+    });
+
+    it('MM-T897_2 - System Console menu footer should not cut off at the bottom', () => {
+        goToAdminConsole();
+
+        // * Scroll to the last item of the page and ensure it can be clicked
+        cy.findByTestId('experimental.bleve').scrollIntoView().click();
+    });
+
+    it('MM-T898 - Individual plugins can be searched for via the System Console search box', () => {
+        // # Uninstall all plugins
+        const uninstallAllPlugins = () => {
+            cy.apiGetAllPlugins().then(({plugins}) => {
+                const {active, inactive} = plugins;
+                inactive.forEach((plugin) => cy.apiRemovePluginById(plugin.id));
+                active.forEach((plugin) => cy.apiRemovePluginById(plugin.id));
+            });
+        };
+
+        goToAdminConsole();
+
+        // # Enable Plugin Marketplace and Remote Marketplace
+        cy.apiUpdateConfig({
+            PluginSettings: {
+                Enable: true,
+                EnableMarketplace: true,
+                EnableRemoteMarketplace: true,
+                MarketplaceUrl: 'https://api.integrations.mattermost.com',
+            },
+        });
+
+        cy.apiInstallPluginFromUrl('https://github.com/mattermost/mattermost-plugin-antivirus/releases/download/v0.1.2/antivirus-0.1.2.tar.gz', true);
+        cy.apiInstallPluginFromUrl('https://github.com/mattermost/mattermost-plugin-autolink/releases/download/v1.2.1/mattermost-autolink-1.2.1.tar.gz', true);
+        cy.apiInstallPluginFromUrl('https://github.com/mattermost/mattermost-plugin-aws-SNS/releases/download/v1.1.0/com.mattermost.aws-sns-1.1.0.tar.gz', true);
+
+        cy.reload();
+
+        // # Type first plugin name
+        cy.get('#adminSidebarFilter').type('Anti');
+        cy.wait(TIMEOUTS.ONE_SEC);
+
+        // * Ensure anti virus plugin is highlighted
+        cy.get('#plugins\\/plugin_antivirus').then((el) => {
```

Comment:
The test fails for me here because of the following error: "Timed out retrying: Expected to find element: #plugins\/plugin_antivirus, but never found it."

This happens because the element is not visible. You may need to do something like:
`cy.get('#plugins\\/plugin_antivirus').scrollIntoView()`

Reply:
Oh snap I didn't realise `apiInstallPluginFromUrl` endpoint being called has a bug which doesn't send out websocket event for downloaded plugins. Since this ticket isn't intended to fix this, I did a work around where I visit town square which updates the state of application then go back to admin console and things work. I've pushed change and try and let me know if works or not.

Reply:
@hahmadia I have your latest changes and I'm getting the following error (see image)
![Screen Shot 2020-10-08 at 11 01 42 AM](https://user-images.githubusercontent.com/487991/95496469-bea0a000-0955-11eb-9577-4ee1fbe08f06.png)

Reply:
@josephbaylon you may need to have the config PluginSettings.EnableUploads set to true in config.json. 
Note: This might require a server restart

Reply:
@josephbaylon ^^ Please try what Rohitesh has said and report back! Thanks

Reply:
@srkgupta @hahmadia That fixed the issue. I'm just wondering if on cypress servers the flag is set to true by default (looks like it is for partial configs but not sure for boot).

Reply:
Looks like it is, https://github.com/saturninoabril/mattermost-cypress-docker/blob/master/.circleci/config.yml#L57 so it's good.